### ISSUE TITLE
Logs page - PatternFly Select component should be used instead of react-select

### ISSUE THREAD

**Issue by mturley** [konveyor/mig-ui#736]
While looking into https://github.com/konveyor/mig-ui/issues/585, I checked for other usages of `react-select` outside the context of the plan wizard. It is used in `LogHeader.tsx`.

I'll probably take care of this in a separate PR, but if someone else does, I'll have a reference for the implementation here in my PR for #585.

**Comment by eriknelson** [comment]
Thanks Mike. Would be nice to have individual small PRs that hit each one.

**Comment by mturley** [comment]
@eriknelson agreed. I hit just the one for the 2nd step of the plan wizard in https://github.com/konveyor/mig-ui/pull/738. It's worth noting, in that PR I landed on a simpler string-based instance of these Select components, but for some of the others (including the Logs one I think) we'll need object values instead of just strings, and we can reference the commit I made before I simplified it: https://github.com/konveyor/mig-ui/pull/738/commits/a6647e0ecd6141900700bb66438d5730362fef13
